README.rst

```diff
@@ -75,8 +75,46 @@ that the tests pass and also the code follows the code style properly. The
 Configuring and Running
 =======================
 
-To be done... integrade is just starting and in the future you can see how to
-configure and run the functional tests in this section.
+Integrade can be configured to test any instance of cloudigrade. The
+**REQUIRED** environment variables are::
+
+    CLOUDIGRADE_BASE_URL # base url without http/https prefix
+    CLOUDIGRADE_TOKEN    # This is the token of the super user created for
+                         # cloudigrade
+
+The **OPTIONAL** environment variables are::
+
+    CLOUDIGRADE_API_VERSION # defaults to 'v1'
+    CLOUDIGRADE_USER        # defaults to 'admin'
+    CLOUDIGRADE_USER_PASS   # at the moment we only use the token to
+                            # authenticate, but this will be useful later
+    USE_HTTPS  # defaults to false so communication is done over http.
+               #  Set to True to use https.
+    SSL_VERIFY # defaults to false. If true make client verify certificate
+
+To run ``cloudigrade`` locally, especially if you want to run a branch that is
+not master, check out that branch and then follow the directions in the
+Cloudigrade readme for `running locally in OpenShift
+<https://github.com/cloudigrade/cloudigrade#running-locally-in-openshift>`_.
+Then, to deploy the code in that branch in particular, follow the directions
+for `deploying in-progress code to OpenShift
+<https://github.com/cloudigrade/cloudigrade#running-locally-in-openshift>`_.
+
+To run ``integrade`` against the test environment, it is necessary to log your
+local ``oc`` (the command line OpenShift client` into the test environment. You
+can do this by logging in through the web UI and in the menu opened by clicking
+on your user name, there is an option to ``Copy Login Command``. Paste this to
+the terminal to log the ``oc`` client into that OpenShift cluster.
+
+In either case, given that your ``oc`` binary is logged into the correct
+OpenShift cluster, you can collect the necessary `CLOUDIGRADE_BASE_URL` by
+inspecting the output of `oc status`.  To create the necessary super user and
+retreive a token for that user, use the cloudigrade make targets `make oc-user`
+and `make oc-user-authenticate`. Then export these values to
```

Comment:
If those make targets returns the string to be added to the env vars then maybe we could suggest to run these commands:

```
$ export CLOUDGRADE_USER="$(make oc-user)"
$ export CLOUDIGRADE_TOKEN="$(make oc-user-authenticate)"
```

Reply:
`make oc-user` is interactive and prompts you to make a user. `make oc-user-authenticate` is also interactive and asks for the username of the super user you created with `make oc-user`.

I've made a script that will set these values if you source it. Could not figure out how to make it work from within the Makefile. About to push change so you can call `source scripts/oc-auth.sh` to set the variables.

Reply:
I am not sure if make will be able to export those variables. It will be run by a child process of the shell so any exported variables won't got upwards only for children processes.